client/src/components/EditCommentPopup.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import Modal from 'react-bootstrap/Modal'
+import Button from 'react-bootstrap/Button'
+import TextArea from './TextArea'
+
+export default function EditCommentPopup({show, onClose, text, rating, comment_id, commentUpdated}) {
+
+    const [editedComment, setEditedComment] = useState({
+        "text" : text,
+        "rate" : rating,
+        "comment_id" : comment_id
+    })
+
+    function handleChange (event) {
+
+        const { value, name } = event.target;
+        setEditedComment((prevValue) => ({ ...prevValue, [name]: value }));
+    }
+
+    async function updateComment () {
+
+        try {
+          const response = await fetch(`/api/comment`, {
+            method: "PUT",
+            headers: { "Content-Type": "application/json" },
+            body: JSON.stringify(editedComment)
+          })
+    
+        } catch (error) {
+          console.error(error.message)
+        }
+    }
+
+    function handleSubmit(event) {
+
+        event.preventDefault();
+        updateComment();
```

Comment:
Whenever a function is labeled as `async` like `updateComment` is, you must handle the promise or else you'll end up with an error complaining about "an uncaught promise"

Any `async` function will inherently return a Promise, and all promises must be handled. i.e. at the very least, you must tell your code what it should do if the Promise fails to resolve.

```suggestion
        updateComment().catch(console.error);
```